### ISSUE TITLE
TextArea - resize control is not keyboard accessible

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7338]
### Expected Behavior
A TextArea with the resize control should be accessible to keyboard users

### Actual Behavior
Currently it is not possible to resize the TextArea via keyboard

### URL, screen shot, or Codepen exhibiting the issue
https://storybook.grommet.io/?path=/story/input-textarea-simple--simple

### Steps to Reproduce

1. 2. 3.

### Your Environment

- Grommet version:
- Browser Name and version:
- Operating System and version (desktop or mobile):

**Comment by unamdev0** [comment]
Hi @jcfilben, Can I try this to fix this issue or has this already been fixed?

**Comment by jcfilben** [comment]
Hi @unamdev0 feel free to work on this, this issue has not been fixed yet

**Comment by jcfilben** [comment]
Closing as not planned. After discussing with the dev group we decided there isn't enough of a precedent in other TextArea components to support keyboard accessible resizing. We can revisit this in the future if desired.
